cmd/atlas/schema.go

```diff
@@ -7,8 +7,8 @@ import (
 // schemaCmd represents the schema command
 var schemaCmd = &cobra.Command{
 	Use:   "schema",
-	Short: "Work with atlas schemas.",
-	Long:  `Provides ability to interact with schema and data source.`,
+	Short: "Work with atlas schemas",
+	Long:  `Interact with the schema and data source`,
```

Comment:
```suggestion
	Long:  "Interact with the schema and data source",
```

Why the inconsistency between the 2 strings?

Reply:
short is when it's a sub command.
long is when you are -h on a specific command.